catalog/ui/catalog-ui-search/src/main/webapp/js/widgets/cesium.circle.js

```diff
@@ -101,16 +100,18 @@ define([
             },
             drawBorderedCircle: function(model) {
                 // if model has been reset
+
                 var modelProp = model.toJSON();
-                if (this.isModelReset(modelProp) || modelProp.lat === undefined || modelProp.lon === undefined) {
-                    return;
-                }
 
                 // first destroy old one
                 if (this.primitive && !this.primitive.isDestroyed()) {
```

Comment:
Not your code, but might as well update this to be `this.destroyPrimitive()` since it does the same thing.